3main

```diff
@@ -63,8 +63,11 @@
 #          AUTO_SWITCH generalized implementetion
 # v=0021 : papampi
 #          Algo identification edits for Algo mining unification
+# v=0022 : LuKePicci
+#          Reworked OC PerfLevel detection, 0algo_id
 
 source ${NVOC}/1bash
+
 nvOC_ver="nvOC v0019-2.1 - Community Release"   # Do not edit this
 nvOC_3main_ver="v0019-2.1.0021"                  # Do not edit this
```

Comment:
I forgot to increment the revision, than should be fine. Please do it for me, I cannot right now.